modules/varscan/1.0/varscan.smk

```diff
@@ -0,0 +1,194 @@
+#!/usr/bin/env snakemake
+
+
+##### ATTRIBUTION #####
+
+
+# Original Author:  Lauren Chong
+# Module Author:    Helena Winata
+# Contributors:     N/A
+
+
+##### SETUP #####
+
+
+# Import package with useful functions for developing analysis modules
+import oncopipe as op
+
+# Setup module and store module-specific configuration in `CFG`
+# `CFG` is a shortcut to `config["lcr-modules"]["varscan"]`
+CFG = op.setup_module(
+    name = "varscan",
+    version = "1.0",
+    subdirectories = ["inputs", "mpileup", "varscan", "maf","outputs"]
+)
+
+# Define rules to be run locally when using a compute cluster
+# TODO: Replace with actual rules once you change the rule names
+localrules:
+    _varscan_input_bam,
+    _varscan_symlink_maf,
+    _varscan_output_vcf,
+    _varscan_output_maf,
+    _varscan_dispatch,
+    _varscan_all,
+
+
+##### RULES #####
+
+# Symlinks the input files into the module results directory (under '00-inputs/')
+rule _varscan_input_bam:
+    input:
+        bam = CFG["inputs"]["sample_bam"]
+    output:
+        bam = CFG["dirs"]["inputs"] + "bam/{seq_type}--{genome_build}/{sample_id}.bam"
+    run:
+        op.relative_symlink(input.bam, output.bam)
+        op.relative_symlink(input.bam + ".bai", output.bam + ".bai")
+
+
+rule _varscan_bam2mpu:
+    input:
+        bam = CFG["dirs"]["inputs"] + "bam/{seq_type}--{genome_build}/{sample_id}.bam",
+        fasta = reference_files("genomes/{genome_build}/genome_fasta/genome.fa")
```

Comment:
Does samtools mpileup require a .fai file? (I've forgotten) if so, it is probably better to explicitly specify it as an input

Reply:
I will symlink the .fai as inputs too